ethereum/ethereum-org-website
# Deploy v2.6.7

***
### :zap: Changes
- Assets page update (#1415) @ryancreatescopy
- Added latest Ben and Danny articles (#1413) @ryancreatescopy
- Added discord support channel name (#1414) @gilbertginsberg
- Added contest form (#1408) @gilbertginsberg
- Remove past events (#1409) @jjmstark
- Typo Fix: Update get-eth.js (#1410) @ethjoe
- Converted How to submit section to h2 tag so it appears in toc (#1407) @gilbertginsberg
- Update Brownie GH url [Fixes #1404] (#1405) @samajammin
- Fix typo in brownie repo (#1402) @samajammin
- wrapped breadcrumbs for mobile (#1388) @ph5500

## :house: Internal

- Medalla data challenge (#1401) @ryancreatescopy

***
### :unicorn: Contributors
Thank you @ethjoe, @gilbertginsberg, @jjmstark, @ph5500, @ryancreatescopy and @samajammin for the contributions! :trophy:
***
